src/Tribe/Views/V2/Assets.php

```diff
@@ -99,6 +99,49 @@ public function register() {
 			]
 		);
 
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
+		// 	$plugin,
```

Comment:
:no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: bcb0686daa7c33c5bf4d7828ed016af8</sub>